bnd.bnd

```diff
@@ -1,4 +1,4 @@
 Sling-Initial-Content: SLING-INF/apps/repl/components/repl;overwrite:=true;path:=/apps/repl/components/repl,\
-    SLING-INF/content/htl/repl.json;overwrite:=true;path:=/content/htl/repl,\
+    SLING-INF/content/htl/repl.json;overwrite:=true;path:=/content/htl,\
```

Comment:
```suggestion
    SLING-INF/content/htl;overwrite:=true;path:=/content/htl,\
```

This seems the safer approach, because otherwise we might rely on a bug in jcr.contentloader.